iris-mpc-common/src/helpers/sqs.rs

```diff
@@ -43,7 +43,7 @@ pub async fn get_next_sns_seq_num(
         }
     } else {
         tracing::info!("Timeout while waiting for a message in the queue. Queue is clean.");
-        Ok(None)
+        Ok(Some(0))
```

Comment:
a bit concerned if 0 is the best value to signal this, can this not be 0 on new queues at least?